clusters.yml

```diff
@@ -3,7 +3,8 @@
   tasks:
 
     - name: Register a cluster
-      register_cluster:
+      clusters:
+        action: present
```

Comment:
This would be `state` instead of action. Can we get rid of data?

Reply:
Fixed in newest commit